packages/design-system/src/styles/components/_Button.scss

```diff
@@ -22,33 +22,35 @@
     border-color: $color-primary-darker;
     color: $color-primary-darker;
   }
+  @if $ds-include-focus-styles {
+    &:focus,
+    &.ds-c-button--focus {
+      background-color: $focus-color;
+      border-color: $color-base;
+      box-shadow: $focus-shadow-link;
+      color: $color-base;
+      outline: 3px solid transparent;
+      outline-offset: 2px;
+    }
 
-  &:focus,
-  &.ds-c-button--focus {
-    background-color: $focus-color;
-    border-color: $color-base;
-    box-shadow: $focus-shadow-link;
-    color: $color-base;
-    outline: 3px solid transparent;
-    outline-offset: 2px;
-  }
-
-  // Removes inner outline on buttons in Firefox.
-  &::-moz-focus-inner {
-    border: 0;
+    // Removes inner outline on buttons in Firefox.
+    &::-moz-focus-inner {
+      border: 0;
+    }
   }
```

Comment:
I think there should be an else here
[original](https://github.com/CMSgov/design-system/pull/510/files#diff-08998df4528f2cb9cfdc571c93ad3ce6L22)

Reply:
For which portion of code? 
it looked to me like this block of code was there before and after the focus styles

```
 &:active,
  &.ds-c-button--active {
    border-color: $color-primary-darkest;
    color: $color-primary-darkest;
  }
```

Reply:
else {
  &:focus,
  &.ds-c-button--focus {
    border-color: $color-primary-darker;
    color: $color-primary-darker;
  }
}